packages/wrangler/src/api/startDevWorker/types.ts

```diff
@@ -187,6 +187,7 @@ export type StartDevWorkerOptions = Omit<StartDevWorkerInput, "assets"> & {
 	};
 	dev: StartDevWorkerInput["dev"] & {
 		persist: string;
+		auth?: AsyncHook<CfAccount>; // redefine without config.account_id hook param (can only be provided by ConfigController with access to wrangler.toml, not by other controllers eg RemoteRuntimeContoller)
```

Comment:
Other controllers won't have access to the wrangler.toml config file so refine this without a hook param on `StartDevWorkerOptions.dev.auth`

Reply:
How will this work if e.g. the remote runtime controller unwraps the hook? Will that not fail?